docs/modules/data-connections/pages/data-connections-configuration.adoc

```diff
@@ -26,21 +30,23 @@ XML::
   <data-connection name="my-mysql-database">
     <type>JDBC</type>
     <properties>
-      <property name="jdbcUrl">jdbc:mysql://mysql.example.org:3306</property>
-      <property name="username">my_user</property>
+      <property name="jdbcUrl">jdbc:mysql://mysql.example.org:3306</property> <1>
+      <property name="username">my_user</property> <2>
       <property name="password">my_password</property>
     </properties>
     <shared>true</shared>
   </data-connection>
   <data-connection name="my-other-database">
     <type>JDBC</type>
     <properties>
-      <property name="jdbcUrl">jdbc:h2:mem:my-other-database</property>
+      <property name="jdbcUrl">jdbc:h2:mem:my-other-database</property> <1>
```

Comment:
We should  not mention h2 in the docs.
It should be either mysql or postgres at this point.

Reply:
The H2 data connection has been removed from the example.